### DISCUSSION TITLE
There are many duplicate font-family styles in css

### DISCUSSION THREAD

**Discussion by yb** [mantinedev/mantine#2217] [Q&A]
Hi all,

I'm developing with Mantine and Next.js and I've noticed a lot of duplicate css in the builded html.

For example, in the [official minimal example](https://github.com/mantinedev/mantine-minimal-next-template), even if there is only one button component on the page, there will be 3 repeated `font-family:-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica, Arial, sans-serif, Apple Color Emoji,Segoe UI Emoji;` in the html.

I'm guessing this might be so the components don't affect each other? But sometimes maybe we just need to inherit from body.

I think if there is a chance to have a switch to turn off this feature to reduce duplicate styles in the html?

Thanks 😄

**Comment by rtivital** [comment]
It is required to make components work without any global styles. You can configure font-family on MantineProvider, it will change all these properties.

**Reply by yb** [reply]
Hi @rtivital, I tried configuring `fontFamily` in the theme property of the MantineProvider, if set to `undefined` it still generate a long font-family css, now I set it to `inherit` to make the css clean.
Is there a way to not generate the `font-family` style?

**Reply by rtivital** [reply]
If you set font-family to undefined, it will consider that there is not any value and will add default value instead. You can set to inherit if you really want that, I do not see any issue with long font-family though, it does not impact performance in any way.
